appserver/appclient/server/core/src/main/java/org/glassfish/appclient/server/core/jws/JWSAdapterManager.java

```diff
@@ -187,7 +187,10 @@ synchronized File rootForSignedFilesInDomain() {
      * @param content
      */
     void addStaticSystemContent(final String lookupURI, StaticContent newContent) throws IOException {
-        systemAdapter().addContentIfAbsent(lookupURI, newContent);
+        AppClientHTTPAdapter adapter = systemAdapter();
```

Comment:
If starting the `systemAdapter` fails `systemAdapter()` logs an error and returns `null` so I figured that this should be handled without throwing an exception to the outside. Error is already logged so we just don't add the entry.